pkgs/development/python-modules/sphinx/default.nix

```
@@ -32,33 +28,35 @@
 , cython
 , html5lib
 , pytestCheckHook
-, typed-ast
 }:
 
 buildPythonPackage rec {
   pname = "sphinx";
-  version = "5.3.0";
-  format = "pyproject";
+  version = "6.1.3";
+  format = "flit";
```

```suggestion
  format = "pyproject";
```

Please roll that back. We are going to deprecate the `flit` format.